ferrocene/doc/safety-manual/src/rustfmt/options.rst

```diff
@@ -0,0 +1,15 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Tool Options
+============
+
+``rustfmt`` is qualified exclusively for the following command line options:
+
+* Users shall pass command line option ``--emit files`` to each invocation of ``rustfmt``.
+* Users shall pass command line option ``--edition 2021`` to each invocation of ``rustfmt``.
+
+.. |Tool| replace:: ``rustfmt``
+.. |Link| replace:: :doc:`user-manual:rustfmt/cli`
+.. include:: /partials/options.rst
```

Comment:
I would not share the snippet across the various tools.

A lot of the text in the snippet for rustc doesn't apply: there are no flags that we restrict in the documentation, I don't think we need to mandate people pass any flags all the time, and these flags don't affect codegen. I think we can get away with a far smaller piece of text here.

Also, this approach for partials is pretty hacky, and I'd prefer a better templating solution if we were to need to share snippets in the future. Let's discuss about that in Zulip.